ts/DataGrid/DefaultOptions.ts

```diff
@@ -33,6 +33,21 @@ import Globals from './Globals.js';
  * */
 
 const DefaultOptions: Globals.DeepPartial<Options> = {
+    accessibility: {
+        enabled: true,
+        cellEditing: {
+            description: 'Editable cell.',
```

Comment:
IMO all these options should be in the `lang` object.

Reply:
I cannot agree, the API is based on solutions in core / main highcharts.

Reply:
I agree with @karolkolodziej. Since the descriptions are well-placed in `accessibility`, the status messages might be better suited in `lang.accessibility`, as shown here: https://api.highcharts.com/highcharts/lang.accessibility. But I may be wrong, so let's think about it some more.